fixtures/all-versions.github

```diff
@@ -402,7 +409,7 @@ jobs:
           echo "HCNUMVER=$HCNUMVER" >> "$GITHUB_ENV"
           echo "ARG_TESTS=--enable-tests" >> "$GITHUB_ENV"
           echo "ARG_BENCH=--enable-benchmarks" >> "$GITHUB_ENV"
-          echo "HEADHACKAGE=false" >> "$GITHUB_ENV"
+          if [ $((! GHCJSARITH && HCNUMVER >= 90800)) -ne 0 ] ; then echo "HEADHACKAGE=true" >> "$GITHUB_ENV" ; else echo "HEADHACKAGE=false" >> "$GITHUB_ENV" ; fi
           echo "ARG_COMPILER=--$HCKIND --with-compiler=$HC" >> "$GITHUB_ENV"
```

Comment:
I had to use 

```
echo "ARG_COMPILER=--ghc --with-compiler=$HC" >> "$GITHUB_ENV"
```

for ghc-exactprint in https://github.com/alanz/ghc-exactprint/commit/36f329fb88cf76538b7385e4681dfa669076f42c#diff-73f8a010e9b95aa9fe944c316576b12c2ec961272d428e38721112ecb1c1a98bR71.

This PR sets it to `--GHC`